pyro/mesh/patch.py

```diff
@@ -34,9 +34,9 @@
 import h5py
 import numpy as np
 
-import pyro.mesh.boundary as bnd
-from pyro.mesh.array_indexer import ArrayIndexer, ArrayIndexerFC
-from pyro.util import msg
+# import pyro.mesh.boundary as bnd
+# from pyro.mesh.array_indexer import ArrayIndexer, ArrayIndexerFC
+# from pyro.util import msg
```

Comment:
These shouldn't be commented out :)
```suggestion
import pyro.mesh.boundary as bnd
from pyro.mesh.array_indexer import ArrayIndexer, ArrayIndexerFC
from pyro.util import msg
```